src/transformers/image_transforms.py

```diff
@@ -365,6 +365,7 @@ def resize(
     # To maintain backwards compatibility with the resizing done in previous image feature extractors, we use
     # the pillow library to resize the image and then convert back to numpy
     do_rescale = False
+    original_type = image.dtype
```

Comment:
It's going to raise the error in case the input image is Pillow Image

```python
>>> Image.new("RGB", (100, 100)).dtype
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Image' object has no attribute 'dtype'
```

Reply:
Thanks, I fixed it.